src/command/run.go

```diff
@@ -7,49 +7,63 @@ import (
 	"strings"
 )
 
+// Options defines optional arguments for ShellRunner.RunWith().
```

Comment:
Thoughts on naming this `RunOptions`? `Options` seems a bit generic in case we have other functions in this folder that will also have options in the future

Reply:
I agree its a bit generic. Which other options do you foresee to exist in the `command` package? I expect it to always just contain logic to run commands in subshells. 

Another concern is that renaming this to `RunOptions` would result in stutter when being used as an argument to a `Run` function (the primary use case for this struct). Example:

```go
RunWith(RunOptions{Dir: dir, Essential, ...
```

Reply:
Not really forseeing anything but just think it would make sense to be a little more specific.

If it was used outside the package if would have the command prefix anyway

```go
command.RunWith(command.Options{Dir: dir, Essential, ...
```

Reply:
One rule of thumb that I use is the principle of least surprise. If there is only one _options_ type in a package, and it's clear to everybody what it means, just name it "options". I think `command.Options` is pretty clear. Just seeing that without additional context, I would assume these are the options for executing a command. If there are multiple options types, name them more specifically, like "RunOptions" and "ResultOptions".

I agree that there is some amount of stutter when using this externally (which we do). But the stutter would be even worse if renamed to "RunOptions":

```go
command.RunWith(command.RunOptions{Dir: dir, Essential, ...